src/plugins/data/public/plugin.ts

```diff
@@ -115,14 +115,19 @@ export class DataPublicPlugin
   private readonly fieldFormatsService: FieldFormatsService;
   private readonly queryService: QueryService;
   private readonly storage: DataStorage;
+  private readonly sessionStorage: DataStorage;
 
   constructor(initializerContext: PluginInitializerContext<ConfigSchema>) {
     this.searchService = new SearchService(initializerContext);
     this.uiService = new UiService(initializerContext);
     this.queryService = new QueryService();
     this.fieldFormatsService = new FieldFormatsService();
     this.autocomplete = new AutocompleteService(initializerContext);
-    this.storage = createStorage({ engine: window.localStorage, prefix: 'opensearch_dashboards.' });
+    this.storage = createStorage({ engine: window.localStorage, prefix: 'opensearchDashboards.' });
```

Comment:
Lets not change the key. Else users stored cache will be wiped out with a version update

Reply:
correct, the original prefix was `opensearchDashboards.` so this was restoring it back to the older key.